src/ert/ensemble_evaluator/evaluator.py

```diff
@@ -271,23 +279,10 @@ async def handle_dispatch(self, websocket: WebSocketServerProtocol) -> None:
                             f"ignoring since I am {self.ensemble.id_}"
                         )
                         continue
-                    try:
-                        if event["type"] == EVTYPE_FORWARD_MODEL_CHECKSUM:
-                            await self.forward_checksum(event)
-                        else:
-                            await self._events.put(event)
-                    except BaseException as ex:
```

Comment:
This should be a separate commit (you choose if also a different PR), to be able to have a commit message properly explaining why.

Reply:
Ok, can remove it from this PR.